### PR TITLE
Updates focus for links and keyboard accessibility

### PR DESCRIPTION
**Pull request by tsidel**
Fixes for #1066

### CODE REVIEW

File: src/_components/footer/Footer.svelte [hunk 1]
```diff
@@ -49,9 +49,9 @@
 							{#each supportLinks as supportLink}
 								<li>
 									<a
-										tabindex="-1"
 										href={supportLink.href}
-										class="text-base text-gray-500 hover:text-that-blue"
+										class="text-base text-gray-500 hover:text-that-bluef ocus:border-thatOrange-700
```
Comment (zaudtke) [inline src/_components/footer/Footer.svelte:53]:
```suggestion
										class="text-base text-gray-500 hover:text-that-blue focus:border-thatOrange-700
```

Comment (csell5) [reply src/_components/footer/Footer.svelte]: nice catch @zaudtke

**Review by zaudtke — DISMISSED** [review]

File: src/_components/footer/Footer.svelte (hunk 1 above)
Comment (tsidel) [reply src/_components/footer/Footer.svelte]: Thanks @zaudtke, fixed it!

File: src/_components/nav/interiorNav/Large.svelte [hunk 2]
```diff
@@ -149,7 +149,7 @@
 	<div class="ml-4 flex items-center justify-center md:ml-6">
 		{#if $hasNotifications}
 			<button
-				class="max-w-xs h-10 w-10 rounded-full text-white focus:outline-none duration-150 ease-in-out hover:bg-thatBlue-500"
+				class="max-w-xs h-10 w-10 rounded-full text-white hover:bg-thatBlue-500"
```
Comment (csell5) [inline src/_components/nav/interiorNav/Large.svelte:152]: Was there a reason to take the animation out?

**Review by csell5 — CHANGES_REQUESTED** [review]

File: src/_components/nav/interiorNav/Large.svelte (hunk 2 above)
Comment (tsidel) [reply src/_components/nav/interiorNav/Large.svelte:152]: No. That could stay in. I think I got too aggressive when I was trying to figure out the culprit early on. I'll add those back in.

**Review by zaudtke — APPROVED** [review]

**Pull request merged**